numpy/ma/core.py

```diff
@@ -6130,8 +6130,11 @@ def __new__(self):
     def __array_finalize__(self, obj):
         return
 
-    def __array_wrap__(self, obj):
```

Comment:
I'm surprised this didn't error already when something tried to pass it a `context`